packages/@uppy/compressor/types/index.d.ts

```diff
@@ -7,6 +7,12 @@ export interface CompressorOptions extends PluginOptions {
   locale?: CompressorLocale
 }
 
+export type GenericEventCallback = () => void;
+
+export interface UppyEventMap {
+  'compressor:complete': GenericEventCallback
```

Comment:
Should we consider giving all compressed files as an argument? I think it makes sense. I actually needed it for better e2e tests as well.

Reply:
Good idea, done!